staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -107,6 +105,10 @@ func NewDefaultKubectlCommand() *cobra.Command {
 func NewDefaultKubectlCommandWithArgs(o KubectlOptions) *cobra.Command {
 	cmd := NewKubectlCommand(o)
 
+	if o.AliasHandler != nil {
+		o.AliasHandler.InjectAll(cmd)
```

Comment:
Does that mean;

User has a plugin `kubectl-foo` installed. 
If user creates an `foo` alias for some builtin command, alias has higher precedence than plugin?